content/docs/08-user-management/04-saml-sso/05-palette-sso-with-onelogin.md

```diff
@@ -0,0 +1,219 @@
+
+18. In order for Palette to map the groups configured in OneLogin, you need to create a Team in Palette. This Team name needs to match with the value that is returned for the groups configured in OneLogin. In this example we'll be creating an **Administrators** team. This also matches with the Administrators value that is assigned to each member of the Administrators group. In Palette, go to **Tenant Settings** --> **Users & Teams** --> **Teams** and select **Create team**. You only need to fill in the Team name and don't have to manually add members. This will happen automatically, because the users will belong to a certain group and that group is mapped to a team with certain permissions in Palette. After filling in the Team name, you can click *confirm*. 
+
+<br />
+
+  ![Create New Team](/oidc-onelogin-images/new-team.png)
+
+<br />
+
+19. Next you need to give the members of this team a set of permissions. In this example we will give all members *Tenant Admin* permissions, since this is the Administrators team. Of course, you can define these permissions fully to your liking. In order to set the correct permissions, you can select the team you just created (**Tenant Settings** --> **Users & Teams** --> **Teams** --> **Your team**), then select **Tenant Roles** --> **Add Tenant Role** and select the **Tenant Admin** box. This will auto-select all other boxes. 
+
+<br />
+
+  ![Add Tenant Role](/oidc-onelogin-images/tenant-admin-role.png)
+
+<br />
+
+Click **Confirm** and you should have a configuration similar to this. 
+
+<br />
+
+  ![Permissions](/oidc-onelogin-images/roles-full.png)
+
+<br />
+
+# Validate
+
+1. The SSO configuration is now complete and it's time to test all of our efforts. You can logout out of Palette and you'll be redirected to a new login screen. 
+
+<br />
+
+  ![Logout](/oidc-onelogin-images/logout.png)
+
+<br />
+
+2. The login screen now defaults to a **Sign In** button. If you are already signed in into OneLogin, once you click on **Sign In** you will be automatically signed in and should have the correct permissions set. If you are unable to sign in, you can always use a username and password. 
+
+<br />
+
+  ![SSO Login Screen](/oidc-onelogin-images/login-screen.png)
+
+<br />
+
+
+3. You can check which teams you've been mapped to, by going to **Tenant Settings** --> **Users & Teams** --> **Users** and you'll directly see which teams you belong to.  
+
+<br />
+
+  ![Check Team Member](/oidc-onelogin-images/team-member.png)
+
+<br />
+
+# Troubleshooting
+With the OpenID Connect Inspector, you can send requests to OneLogin and check what is sent in the payload. This way you can make sure that you are using the right claims and scopes. When using this, you will need to add the Inspector callback URL to your applications Redirect URIs list. 
+[OpenID Connect Inspector Tutorial](https://youtu.be/do0agd71hE8)
+[OpenID Connect Inspector](https://developers.onelogin.com/openid-connect/inspector)
```

Comment:
```suggestion
If you encounter issues while setting up OIDC, you can use OneLogin's [OpenID Connect Inspector](https://developers.onelogin.com/openid-connect/inspector) tool to help you debug. You can use the tool to send requests to OneLogin and check what is sent in the payload. Use the tool to verify you are using the correct claims and scopes. When using this, you will need to add the Inspector callback URL to your application Redirect URIs list. 
[OpenID Connect Inspector Tutorial](https://youtu.be/do0agd71hE8)
```
Language cleanup.

For adding a Youtube video, use the video component. Refer to this part of the [README](https://github.com/spectrocloud/librarium#youtube-video) for example and usage.